### PR TITLE
Core 36764 - Populate spec for components/Personalization/helper/dom/getParent.js

### PR DESCRIPTION
**Pull request by ninaceban**
## Description
populate getParent spec

## Motivation and Context

populate getParent spec
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have submitted a [documentation](https://github.com/AdobeDocs/alloy-docs) pull request or no changes are needed.
- [x] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: test/unit/specs/components/Personalization/helper/dom/getParent.spec.js [hunk 1]
```diff
@@ -10,5 +10,32 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
+
+  it("should return the parent node if exists", () => {
+    const parentNode = createNode("div", { id: "parentId" });
+    const childNode = createNode("div", { id: "childId" });
+
+    appendNode(parentNode, childNode);
+    appendNode(document.head, parentNode);
+
+    const result = getParent(getElementById("childId"));
+
```
Comment (artur-ciocanu) [inline test/unit/specs/components/Personalization/helper/dom/getParent.spec.js]: I think this is redundant, you already check the `tagName`.

File: test/unit/specs/components/Personalization/helper/dom/getParent.spec.js [hunk 2]
```diff
@@ -10,5 +10,32 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
-// eslint-disable-next-line no-unused-vars
 import getParent from "../../../../../../../src/components/Personalization/helper/dom/getParent";
+import {
+  selectNodes,
+  removeNode,
+  appendNode,
+  createNode
+} from "../../../../../../../src/utils/dom";
+import { getElementById } from "../../../../../../../src/components/Personalization/helper/dom";
+
```
Comment (artur-ciocanu) [inline test/unit/specs/components/Personalization/helper/dom/getParent.spec.js]: Probably this should be rename to `Personalization::helper::dom::getParent`

Comment (ninaceban) [reply test/unit/specs/components/Personalization/helper/dom/getParent.spec.js]: thank you

File: test/unit/specs/components/Personalization/helper/dom/getParent.spec.js (hunk 1 above)
Comment (ninaceban) [reply test/unit/specs/components/Personalization/helper/dom/getParent.spec.js]: thank you

**Pull request merged**